packages/frontend/web/src/components/input.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+type InputProps = {
+  readonly type: 'email' | 'password' | 'text';
+  readonly placeholder: string;
+};
+
+export default function Input({ type, placeholder }: InputProps) {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Tu peux
```suggestion
  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
```
et importer ChangeEvent